tests/unit/Storefront/Theme/Validator/SCSSValidatorTest.php

```diff
@@ -173,7 +173,28 @@ public static function sanitizeDataProvider(): \Generator
             ],
             'rgba(4, 4, 4, 0.5)',
         ];
-        // incorrect but valid(no error in compilation)
+        // Empty values (are valid and will be set to null)
```

Comment:
could you also test cases, where the value `0` or `'0'` are allowed? if I understand your changes correctly, those would have failed before

Reply:
Thx, @mitelg, I added test cases for "0" values.